spec/controllers/equipment_models_controller_spec.rb

```diff
@@ -1,421 +1,280 @@
-        FactoryGirl.create(:reservation, equipment_model: model,
-                                         start_date: Time.zone.today + 10.days,
-                                         due_date: Time.zone.today + 12.days)
-        get :show, id: model
-        expect(assigns(:pending)).to eq([] << res_starting_today <<
-                                            res_starting_this_week)
-      end
-    end
-
-    context 'with non-admin user' do
+  describe 'GET index' do
+    shared_examples_for 'GET index success' do |user_role|
       before do
-        sign_in FactoryGirl.create(:user)
-        get :show, id: model
-      end
-
-      it_behaves_like 'GET show success'
-    end
-
-    context 'availability calendar' do
-      before(:each) do
-        @model = FactoryGirl.create(:equipment_model)
-        2.times do
-          FactoryGirl.create(:equipment_item, equipment_model: @model)
-        end
-        sign_in FactoryGirl.create(:user)
-      end
-
-      shared_examples_for 'calculates availability correctly' do |expected|
-        it do
-          get :show, id: @model
-          day_data = assigns(:avail_data).select do |d|
-            d[:start] == Time.zone.today + 1.day
-          end
-          expect(day_data.first[:title]).to eq(expected.to_s)
-        end
-      end
-
-      context 'with no reservations' do
-        it_behaves_like 'calculates availability correctly', 2
+        mock_user_sign_in(mock_user(user_role))
       end
 
-      context 'with upcoming request' do
-        before do
-          FactoryGirl.create(:request,
-                             equipment_model: @model,
-                             start_date: Time.zone.today + 1.day,
-                             due_date: Time.zone.today + 1.day)
-        end
-
-        it_behaves_like 'calculates availability correctly', 2
-      end
-
-      context 'with returned reservation' do
-        before do
-          FactoryGirl.create(:checked_in_reservation,
-                             equipment_model: @model,
-                             start_date: Time.zone.today - 1.day,
-                             due_date: Time.zone.today + 1.day)
-        end
-
-        it_behaves_like 'calculates availability correctly', 2
+      describe 'basic function' do
+        before { get :index }
+        it_behaves_like 'successful request', :index
       end
 
-      context 'with upcoming reservation' do
-        before do
-          FactoryGirl.create(:valid_reservation,
-                             equipment_model: @model,
-                             start_date: Time.zone.today + 1.day,
-                             due_date: Time.zone.today + 1.day)
+      context 'with @category set' do
+        it 'populates an array of of active category-type equipment models' do
+          cat = mock_category(id: 1)
+          models = mock_eq_models(traits: [[:with_category, cat: cat]])
+          stub_collection_methods models
+          expect_any_instance_of(Array).to \
```

Comment:
actually, I think this structure makes more sense, because the mock setups are actually different for each context:

```ruby
it 'defaults to all active equipment models' ...

context 'with @category set' do
  it 'restricts results to category' ...
end 
context 'with show deleted' do
  it 'includes deactivated models' ...
end
```
testing that category can be restricted when deactivated models are shown seems redundant